second_cplex/second_cplex.mod

```diff
@@ -34,14 +37,30 @@ range day3 = 13..18;
 range day4 = 19..24;
 range day5 = 25..30;
 range r4 = 1..N4;
+int basePrepTime = 240;
+int totalTime = 1500;
+
+float totalTeacherMin[r2];
+float prep[r2];
+float teachMin[r2];
+
+execute {
+
+	for(var j in r2)
+	{
+	totalTeacherMin[j]= FTE[j] * totalTime;
+	prep[j] = FTE[j]*basePrepTime; //prep time allocation
+	teachMin[j] = totalTeacherMin[j]-prep[j]; //teaching minute allocation
+	}
 
+}
 // Problem parameters are defined below
-int prep[r2]= [240,240,120]; //prep time array
 int lengtht[r4]= [40,60,50,50,60,40,
 				 40,60,50,50,60,40,
 				 40,60,50,50,60,40,
 				 40,60,50,50,60,40,
 				 40,60,50,50,60,40];   
+
 int teachMin[r2]=[1260,1260,630];
```

Comment:
dont see this array being used anymore because there is more teachers, this can be deleted?

Reply:
int teachMin[r2]=[1260,1260,630];
float FTE[r2]= [1.0,1.0,0.5];
float totalTeacherMin[r2];
int totalTime = 1500;

All need to be deleted as they are previously defined